src/modules/configs/config.py

```diff
@@ -0,0 +1,193 @@
+from logging import Logger
+from json import loads
+from typing import Union
+import os
+
+from jsonschema import validate
+
+from .schemas import application, project, operator
+from ..global_vars import global_vars
+from .templates import templates
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+        if self.validate_conf(conf) is None:
+            return conf
+        return False
+
+    def validate_conf(self, conf: dict) -> None:
+        if self.conf_type == "application":
+            schema_file = application
```

Comment:
Thinking about something like `schema_file = importlib.import_module(f"schemas.{self.conf_type}")` but that's not very clean (need to read through this first https://stackoverflow.com/questions/477096/python-import-coding-style).

Reply:
For now maybe `schema_file = self.get_schema_file(self.conf_type)` ?

Reply:
Better yet: add schema_file as Conf attribute and provide it later from specific classes e.g.
```
class OperatorConf(Conf):
    """Class for operator configuration files"""

    def __init__(self, conf_path: Union[str, bool]):
        super().__init__(conf_path, "operator", schemas.operator)
```